docs/architecture-diagrams.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python3
+"""
+Generate architecture diagrams for the documentation
+"""
+from diagrams import Diagram, Edge, Cluster
+from diagrams.k8s.compute import Deployment, Pod
+from diagrams.k8s.network import Service
+from diagrams.k8s.storage import PVC
+
+IMAGES_DIR = "source/_static/images"
+
+with Diagram("JupyterHub + SSH + SFTP", show=False, outformat="png", filename=f"{IMAGES_DIR}/architecture") as diag:
```

Comment:
It would be good to clarify this is assuming we have deployed jupyterhub ssh alongside z2jh.